README.md

```diff
@@ -1822,6 +1822,7 @@ The following configuration is available in `disco_elasticache.ini` to configure
     port=6379
     parameter_group=default.redis2.8
     num_nodes=2
+    maintenance_window=sun:5:00-sun:09:00
```

Comment:
Jeannie mentioned that saturday 1am to 2am would be good. So I've assigned the appropriate UTC time for this.